extensions/kill-process/src/index.tsx

```diff
@@ -117,20 +172,46 @@ export default function ProcessList() {
           return (
             <List.Item
               key={index}
-              title={process.name}
+              title={(process.type == "aggregatedApp" ? process.appName : process.processName) ?? ""}
               subtitle={subtitleString(process)}
               icon={icon}
-              accessoryTitle={`${process.cpu}%`}
+              accessoryTitle={
+                sortByMem
+                  ? `${process.cpu.toFixed(2)}% ${prettyBytes(process.mem * 1024)}`
+                  : `${prettyBytes(process.mem * 1024)} ${process.cpu.toFixed(2)}%`
+              }
               actions={
                 <ActionPanel>
-                  <ActionPanel.Item title="Kill" icon={Icon.XmarkCircle} onAction={() => killProcess(process)} />
-                  {copyToClipboardAction(process)}
-                  <ActionPanel.Item
+                  <Action title="Kill" icon={Icon.XMarkCircle} onAction={() => killProcess(process)} />
+                  {process.path == null ? null : <Action.CopyToClipboard title="Copy Path" content={process.path} />}
+                  <Action
                     title="Reload"
                     icon={Icon.ArrowClockwise}
                     shortcut={{ key: "r", modifiers: ["cmd"] }}
                     onAction={() => fetchProcesses()}
                   />
+                  <Action
+                    title={`Sort by ${sortByMem ? "CPU" : "memory"} usage`}
+                    icon={Icon.Filter}
+                    shortcut={{ key: "tab", modifiers: [] }}
+                    onAction={() => {
+                      setSortByMem(!sortByMem);
+                      showToast({
```

Comment:
I'm not 100% sure about changing these flags from actions. The reason for my uncertainty is that these don't actually change the extension's preference, they just update flags for this session, which might be confusing to the user. @pernielsentikaer, what do you folks think?

Reply:
I can see why it could be nice to be able to change this on the fly but then I don't think there should be anything in preferences and then maybe use `useCachedState` to cache the values between loads